src/collective/cover/tiles/carousel.py

```diff
@@ -66,12 +66,21 @@ def populate_with_object(self, obj):
         :type uuids: List of strings
         """
         super(ListTile, self).populate_with_object(obj)  # check permission
-        uuids = ICoverUIDsProvider(obj).getUIDs()
+        if obj.portal_type in ('Topic', 'Collection'):
+            uuids = [i.UID for i in obj.queryCatalog()]
+        else:
+            uuids = [self.get_uuid(obj)]
         # accept just elements with a lead image
         uuids = [i for i in uuids if self._has_image_field(uuidToObject(i))]
         if uuids:
             self.populate_with_uuids(uuids)
 
+    @view.memoize
+    def accepted_ct(self):
```

Comment:
this tile should in fact accept any content type as even Dexterity folders can have an image field using a behavior; the validation must be done in the `populate_with_object()` method.